src/base/staticClient.ts

```diff
@@ -212,6 +213,8 @@ async function runFetch(
       ...options.client.clientConfig.retrySettings,
       ...options.retrySettings,
     },
+    ...options.client.clientConfig.fetchOptions,
+    ...options.fetchOptions,
```

Comment:
These should be first, rather than last, so that the user can't override the behavior that we want to control. (e.g. if the user specifies a `method`, we don't want that to have any effect, because we know that it won't work.)